packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2925,11 +2939,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         } else if (this.summaryManager !== undefined) {
             return this.summaryManager.enqueueSummarize(...args);
         } else {
-            // If we're not the summarizer, and we don't have a summaryManager, we expect that
-            // generateSummaries is turned off. We are throwing instead of returning a failure here,
-            // because it is a misuse of the API rather than an expected failure.
-            throw new UsageError(
-                `Can't summarize, disableSummaries: ${this.summariesDisabled}`,
+            // If we're not the summarizer, and we don't have a summaryManager, we are
+            // generating an error.
+            throw new Error(
+                `Couldn't find a summarizer`,
             );
```

Comment:
I think UsageError was there for a reason - it's a user error in a sense of using this API when user told us summaries are disabled. We should use right error types as they will get to telemetry, and we should differentiate our bugs from usage errors.